Woof/Woof/Helper/DataTrasnformer/DataTransformer.swift

```diff
@@ -29,15 +29,25 @@ enum DataTransformer {
             .capitalized
     }
 
-    /// Converts a numeric value to an integer within the range of 0 to 5, following the zero-to-five style.
+    /// Converts a numeric value to a rating corresponding to a 5-star rating.
     ///
     /// - Parameter value: The numeric value to be converted.
-    /// - Returns: An integer value within the range of 0 to 5.
-    /// If the value is less than 0, 0 is returned.
-    /// If the value is greater than 5, 5 is returned.
-    /// Otherwise, the rounded value is returned. The rounding rule is schoolbook rounding.
-    static func normalizeToZeroToFiveRange(_ value: Double) -> Int {
+    /// - Returns: The rating from 1 to 5 stars from a 5-star rating, or information that the rating is unavailable.
+    /// If the value is less than or equal 0, returned `.unavailable` to indicate an unavailable rating.
+    /// If the value is between 1 and 5 (inclusive), returned the rating from 1 to 5 stars from a 5-star rating.
+    /// The rounding rule is schoolbook rounding.
+    ///
+    /// /// Example usage:
```

Comment:
Should we provide this?( I think, we can not indicate this, because the rule in mentioned above in the documentation).
If you think it's necessary, why not described another two cases for value = 3.2, 3.5?

Reply:
This example shows how to use this function, not the rounding rule.

Reply:
Up to you, for me it's extra documentation.